### DISCUSSION TITLE
Deprecated tag not shown

### DISCUSSION THREAD

**Discussion by Amorim33** [scalar/scalar#1050] [Q&A]
## Description

We are using Hono for the API, and [Hono Zod Openapi](https://hono.dev/snippets/zod-openapi) to generate the OpenAPI specs.

When deprecating a field, `Scalar` does not render the `deprecated` tag.

![image](https://github.com/scalar/scalar/assets/42624869/e3e6199e-aaa6-461a-b09b-85ae23fc66ae)

## Expected behavior
When a field is deprecated, it is expected to be explicit in the docs.

### Redoc (example)
![image](https://github.com/scalar/scalar/assets/42624869/f017d6b1-727d-443c-a675-24af0cf2cc8d)

- Does it make sense?

## Steps to reproduce

- Add `@scalar/hono-api-reference` to the project;
- Add OpenAPI fields to the OpenAPI json schema.

### Package version
```json
{
    "@scalar/hono-api-reference": "^0.3.56",
}
```
### OpenAPI fields
```json
{
  "additionalAttributes": {
    "type": "object",
    "nullable": true,
    "additionalProperties": {
        "nullable": true
    },
    "description": "Atributos adicionais do item segurado. **O campo será removido futuramente, utilize o campo metadata**.",
    "deprecated": true
  },
}
```

**Comment by Amorim33** [comment]
If this aligns with the library's goals, I'm fully committed to resolving this issue. 🫡

**Reply by hanspagel** [reply]
This aligns with the library’s goals! :D But it’s design relevant, so I’ll leave this for @cameronrohani

**Reply by Amorim33** [reply]
No problem!! Thank you for the feedback!🙏

**Comment by Amorim33** [comment]
Done https://github.com/scalar/scalar/pull/1075
